Hai Reveal conversations: split up text walls, simplify dialog, and fill minor plot holes in mission [B13] Freedom (#7697)
endless-sky/endless-sky: data/hai/hai reveal 5 philosophers.txt

		log `Factions` `Elenchus` `Formerly a pirate stronghold, this city built by generations of kidnapped specialists is a hundred twenty years old now. Humans, Hai, and even Korath live peacefully, working side-by-side, and most have no desire to leave. They value their independence and have refused Republic membership. This city orbits a strange world that defies analysis in a region of space untouched by civilization.`
		log `Liberated Elenchus.`
		conversation

> 			`General Choot'k cuts open the doors of a large hangar, and your ships land. The two hundred operatives pour out wearing armor-plated exoskeletons. They storm through the complex and you follow in their wake. Their guns melt away meter-thick walls in seconds, and most of their opposition lacks protection against the suits' stun gas bombs. The pirates outnumber the Syndicate's operatives twenty to one, but this huge advantage in equipment, combined with training and discipline, gives the operatives an overwhelming advantage in constrained spaces.`

Author: No korath rifles.

> 			`General Choot'k cuts open the doors of a large hangar, and your ships land. The two hundred operatives pour out wearing armor-plated exoskeletons. They storm through the complex and you follow in their wake. Their guns melt away meter-thick walls in seconds, and most of their opposition lacks protection against the suits' stun gas bombs. The pirates outnumber the Syndicate's operatives twenty to one, but this huge advantage in equipment, combined with training and discipline, gives the operatives an overwhelming advantage in constrained spaces.`

Reviewer: `Their guns melt away meter-thick walls in seconds, and most of their opposition lacks protection against the suits' stun gas bombs.` what guns then?

> 			`General Choot'k cuts open the doors of a large hangar, and your ships land. The two hundred operatives pour out wearing armor-plated exoskeletons. They storm through the complex and you follow in their wake. Their guns melt away meter-thick walls in seconds, and most of their opposition lacks protection against the suits' stun gas bombs. The pirates outnumber the Syndicate's operatives twenty to one, but this huge advantage in equipment, combined with training and discipline, gives the operatives an overwhelming advantage in constrained spaces.`

Author: These aren't rifles. The big mech suits. Humans have plasma cannons and ship-mountable flamethrowers (that somehow work in space?) which they could have miniaturized. These suits are clearly way beyond ordinary ground combat tech.

> 			`General Choot'k cuts open the doors of a large hangar, and your ships land. The two hundred operatives pour out wearing armor-plated exoskeletons. They storm through the complex and you follow in their wake. Their guns melt away meter-thick walls in seconds, and most of their opposition lacks protection against the suits' stun gas bombs. The pirates outnumber the Syndicate's operatives twenty to one, but this huge advantage in equipment, combined with training and discipline, gives the operatives an overwhelming advantage in constrained spaces.`

Author: If Korath rifles are involved in this fight, I'll have to completely rewrite the fight.

> 			`General Choot'k cuts open the doors of a large hangar, and your ships land. The two hundred operatives pour out wearing armor-plated exoskeletons. They storm through the complex and you follow in their wake. Their guns melt away meter-thick walls in seconds, and most of their opposition lacks protection against the suits' stun gas bombs. The pirates outnumber the Syndicate's operatives twenty to one, but this huge advantage in equipment, combined with training and discipline, gives the operatives an overwhelming advantage in constrained spaces.`

Author: >  I really did not understand that, the text needs clarification then thats why they don't get slaughtered now it makes sense

This is in the text:

> Syndicate operatives pour out wearing armor-plated exoskeletons

Is this not clear enough? Is the confusion that it doesn't say there is a robotic mechanism underneath, carrying the exoskeleton? I don't know what you want me to add.

> 			`General Choot'k cuts open the doors of a large hangar, and your ships land. The two hundred operatives pour out wearing armor-plated exoskeletons. They storm through the complex and you follow in their wake. Their guns melt away meter-thick walls in seconds, and most of their opposition lacks protection against the suits' stun gas bombs. The pirates outnumber the Syndicate's operatives twenty to one, but this huge advantage in equipment, combined with training and discipline, gives the operatives an overwhelming advantage in constrained spaces.`

Reviewer: I don't want them to be involved I just assumed they were because it wasnt clear enough

> 			`General Choot'k cuts open the doors of a large hangar, and your ships land. The two hundred operatives pour out wearing armor-plated exoskeletons. They storm through the complex and you follow in their wake. Their guns melt away meter-thick walls in seconds, and most of their opposition lacks protection against the suits' stun gas bombs. The pirates outnumber the Syndicate's operatives twenty to one, but this huge advantage in equipment, combined with training and discipline, gives the operatives an overwhelming advantage in constrained spaces.`

Author: > mechanized battle suits with armor-plated exoskeletons and weapons larger than a human could carry

Does this work?

> 			`	The squads move room to room, neutralizing all opposition with ease until their opponents switch tactics. The pirates begin using electromagnetic pulse devices to shut down the suits (and all other electronics nearby), leaving affected operatives helpless for ten seconds. Here some of General Choot'k's Unfettered show their strength, darting between operatives and bounding off walls with pulse rifles and long knives. They strike rapidly and withdraw as the operatives suits re-engage, preventing the pirates from coordinating a counter-assault.`

Reviewer: ```suggestion
			`	The squads move room to room, neutralizing all opposition with ease until their opponents switch tactics. The pirates begin using electromagnetic pulse devices to shut down the suits (and all other electronics nearby), leaving affected operatives helpless for ten seconds. Here some of General Choot'k's Unfettered show their strength, their armor seemingly unaffected by it. They begin darting between operatives and bounding off walls with pulse rifles and long knives. They strike rapidly and in an organized manner, before withdrawing as one when their suits seem to run out, covering each others and keeping the enemy in check as the operatives suits re-engage, preventing the pirates from coordinating a counter-assault.`
```
So, I'd imagine they would have suits as well (this seems to imply they dont) but lighter, and with an in built battery to keep it functioning if shot by ion (Hai heritage).

As for the blades, sure, makes sense.

For context, my idea is that Unfettered train a lot in their life. These are the best of the best of them, so they'd have been trained all their life. Maybe add a small note to how well organized they are, how they move as one

Author: The text is too long already. I don't need to add details about the excellent training of the Unfettered.

Reviewer: I feel like the player just needs more possibilities to interact with choices so its less of a text wall
I don't mind it not being added but they Unfettered being portrayed as cowards is what bothers me, especially their elite

> 			`	After a bloody struggle, forty-three surviving operatives win the fight, clearing the way to a blast door. Only a couple dozen operatives are critically or fatally wounded, but the pirates were very good at trapping the suits. The Unfettered fared comparatively better, having mostly acted as reserve and flanking forces; still, over a dozen of them will take their own tails home as trophies after a pirates used miner's cutting power-arm extensions to great effect.`

Reviewer: ```suggestion
			`	After a bloody struggle, forty-three surviving operatives win the fight, clearing the way to a blast door. Only a couple dozen operatives are critically or fatally wounded, but the pirates were very good at trapping the suits. The Unfettered fared comparatively better, having mostly acted as reserve and flanking forces, and being evidently better trained and acting together, supporting those unfavorable positions and withdrawing when the time is right; still, over a dozen of them will take their own tails home as trophies after a pirates used miner's cutting power-arm extensions to great effect.`
```
Doesnt have to be this exactly but I find it weird they're described as being in reserve like this and their little casualties being only linked to that

Author: The number of Syndicate operatives that are "casualties" compared to Unfettered is also inconsistent with the other statement that only a few of them were injured, and most are trapped. I think NV lost track of her casualty rates.

Reviewer: they were outnumbered 1/12 so you'd expect casualties

			`	The blast door that the force now stands before is twelve meters high and wide, and scanners show it to be six meters deep. It is designed to be closed from this side though, and has not been sealed well against being reopened from this side. Soon it grinds part-way open, evidently sabotaged, and the operatives put on their goggles and pass through into the darkness of the next room.`
			`	You could be forgiven for hesitating to follow them, but General Choot'k appears beside you and places one strong paw on your shoulder. "Come, now we discover if our task is finished, or impossible. It is time for bravery." He guides you in after the operatives along with his own troops.`
			choice
				`	"Lead on."`
					goto leadon
				`	"Are you calling me a coward? If I had more time, I'd challenge you to a duel for that."`

> 			`	"Heh heh," he laughs in a low voice. "Maybe soon."`

Reviewer: ```suggestion
			`	"Heh heh," he laughs in a low voice. "Maybe soon."`
			action
				set "unfettered: Choot'k duel"
```
I want to keep track of that!

					to display
						not "Hai Reveal [B13] Freedom: label cadmium"
				`	"What can you tell me about this city?"`
					goto place
					to display
						not "Hai Reveal [B13] Freedom: label place"
				`	"What do you do here?"`
					goto do
					to display
						not "Hai Reveal [B13] Freedom: label do"
						or
							has "Hai Reveal [B13] Freedom: label place"
							has "Hai Reveal [B13] Freedom: label home"
				`	"What do you know about this area of space?"`
					goto space
					to display
						not "Hai Reveal [B13] Freedom: label space"
						has "Hai Reveal [B13] Freedom: label place"
				`	"How did the Korath get here?"`
					goto korath
					to display
						not "Hai Reveal [B13] Freedom: label korath"
				`	"Can you speak to other Korath for me?"`
					goto "translation services"
					to display
						not "Hai Reveal [B13] Freedom: label translation services"
						has "Hai Reveal [B13] Freedom: label korath"
				`	"I wish you a successful future with your newfound freedom. For now, it is time for me to depart."`
			`	"It was a pleasure meeting you," says the human leader.`
				goto leave
			label korath
			action
				set "Hai Reveal [B13] Freedom: label korath"

> 			`	The Korath leader speaks, with the Hai translating. "Several generations ago, three of our raiding ships jumped to the Alcyone system. Numerous pirate vessels left Stormhold and disabled our ships. A large human ship boarded our flagship while crews on the other two vessels listened over communicators. All we heard was screaming. The pirates used deadly gas - what you call 'nerve gas' - to kill the entire crew. The death is excruciatingly painful to Korath. Not wanting that fate, most of the raiders on the other Korath ships mutinied and surrendered. We worked as slaves for the pirate gangs, but now we're here, safe, and free. We are not specialists or their descendants like the others here, but we help as we can.`
> 			`	"The pirates must defend this planet against a space-based lifeform that attacks in flocks. Our ships gave them a significant advantage. Fortunately, we have one more left defend against any threats." The Korath eyes the Republic operatives as he waits for the translator to catch up.`

Reviewer 2: Yes, "see the screaming" is deliberate ;-)

```suggestion
			`	The Korath leader speaks, with a thick, sibilant accent. "Several generations ago, three of our raiding ships jumped to the Alcyone system. Numerous pirate vessels left Stormhold and disabled our ships. A large human ship boarded our flagship, and used what you call 'nerve gas' to kill the entire crew. The death is excruciatingly painful to Korath. The pirates broadcast the footage over to us, so we could see the screaming. Not wanting that fate, most of the raiders on the other Korath ships surrendered. We worked as slaves for the pirate gangs, but now we're here, safe, and free. We are not specialists or their descendants like the others here, but we have other talents.`
			`	"The pirates must defend this planet against a space-based lifeform that attacks in flocks. Our ships gave them a significant advantage. Fortunately, we have one more left to defend against any threats."`
```

Author: The text already explicitly states that some have learned the language. However, there is no indication yet that they are physically capable of making the required sounds. Showing Korath that can speak the language is a major story change.

Author: I think I made your other changes.

			`	"I must ask you keep the presence of Korath here a secret. If the Archons or other Exiles hear of this place, they may attack to retrieve us, or the Exiles may simply pillage. We will remain here in hiding unless the Exiles finally learn to settle on their planet."`
				goto questions
			label "translation services"
			action

> 				set "Hai Reveal [B13] Freedom: label translation services"
> 			`	The Korath leader hisses and replies while the Hai translates. "No, we cannot. Few among the Korath know your language, and we must remain here in secret. Perhaps in time there will be a way, but not yet."`

Reviewer 2: ```suggestion
				set "Hai Reveal [B13] Freedom: label translation services"
			`	The Korath leader hisses. "No, we cannot. We must remain here in secret. Perhaps in time there will be a way, but not yet."`
```

Author: The text already explicitly states that some have learned the language. However, there is no indication yet that they are physically capable of making the required sounds. Showing Korath that can speak the language is a major story change.

			`	The Korath leader looks to the Hai, who responds, "Is that how you were led back here?"`
			`	"Indirectly," you reply.`
			`	The Hai leader continues, "Fascinating, we always wondered if it would pique someone's attention! Our Korath friends here brought a ship with remarkable technology called a "Systems Core". None of the surviving Korath had the knowledge to replicate it, but they were asked to try and many of them have an uncanny knack for engineering. With their help, we built some of our own in a fashion. Being not quite right, they work, but they fuse irrevocably with the ship, and some flaw in their function causes cadmium particulate, which is shed by the system maintenance processes, into the air where it is recirculated and ignored by the system intended to filter it out, since it's part of it."`
			`	The human leader adds, "We worked out how to reduce that effect to safe levels, but kept that information to ourselves in the hope that it would draw some attention. It tended to make the pirates sick more often than anyone else, which also seemed to mean that more of their victims escaped."`
				goto questions
			label home
			action

> 				set "Hai Reveal [B13] Freedom: label home"
> 			`	The three leaders laugh. The Hai answers, "We are home. Most of us were born here, or have lived most of our adult lives here. This is a place of learning, where three races join as one to progress together. Even some of the more recent arrivals, several had few connections back in Hai space and have viewed this place as a grand new novelty. An adventure without compare. Some recent kidnapping victims wish to leave of course, and we will help you find them and send them home."`
> 			`	The Korath speaks next, and the Hai translates. "Where my people came from, everyone was starving, barely subsisting. We had to raid for supplies, or slave away in bleak factory stations. This is a much better place for us."`

Reviewer 2: ```suggestion
				set "Hai Reveal [B13] Freedom: label home"
			`	The three leaders laugh. The Hai answers, "We are home. Most of us were born here, or have lived most of our adult lives here. This is a place of learning, where three races join as one to progress together. Even some of the more recent arrivals, several had few connections back in Hai space and have viewed this place as a grand new novelty. An adventure without compare. Some recent kidnapping victims wish to leave of course, and we will help you find them and send them home."`
			`	The Korath speaks next. "Where my people came from, everyone was starving, barely subsisting. We had to raid for supplies, or slave away in bleak factory stations. This is a much better place for us."`
```

Author: The text already explicitly states that some have learned the language. However, there is no indication yet that they are physically capable of making the required sounds. Showing Korath that can speak the language is a major story change.

		log `Factions` `Elenchus` `Formerly a pirate stronghold, this city built by generations of kidnapped specialists is a hundred twenty years old now. Humans, Hai, and even Korath live peacefully, working side-by-side, and most have no desire to leave. They value their independence and have refused Republic membership. This city orbits a strange world that defies analysis in a region of space untouched by civilization.`
		log `Liberated Elenchus.`
		conversation

> 			`General Choot'k cuts open the doors of a large hangar, and your ships land. The two hundred Syndicate operatives pour out wearing mechanized battle suits with armor-plated exoskeletons and weapons larger than a human could carry. They storm through the complex and you follow in their wake. Their guns melt away meter-thick walls in seconds, and most of their opposition lacks protection against the suits' stun gas bombs. The pirates outnumber the Syndicate's operatives twenty to one, but this huge advantage in equipment, combined with training and discipline, gives the operatives an overwhelming advantage in constrained spaces.`

Reviewer: ```suggestion
			`General Choot'k cuts open the doors of a large hangar, and your ships land. The two hundred Syndicate operatives pour out wearing mechanized battle suits with weapons larger than a human could carry. They storm through the complex and you follow in their wake. Their guns melt away meter-thick walls in seconds, and most of their opposition lacks protection against the suits' stun gas bombs. The pirates outnumber the Syndicate's operatives twenty to one, but this huge advantage in equipment, combined with training and discipline, gives the operatives an overwhelming advantage in constrained spaces.`
```
dont think we need to add that? I mean if its a mechanized battle suit it for sure has armor? idk though

Author: No, a mech suit can be unarmored. Besides, explicitly saying "armor-plated exoskeletons" emphasizes how durable they are.

Reviewer: Sounds good to me, you can resolve this (always the case when I put a thumbs up)

> 			`General Choot'k cuts open the doors of a large hangar, and your ships land. The two hundred Syndicate operatives pour out wearing mechanized battle suits with armor-plated exoskeletons and weapons larger than a human could carry. They storm through the complex and you follow in their wake. Their guns melt away meter-thick walls in seconds, and most of their opposition lacks protection against the suits' stun gas bombs. With pirates focusing their fire on the imposing suits, the Unfettered make many attacks of opportunity, inflicting massive casualties. The pirates outnumber the attackers ten to one, but this huge advantage in equipment, training, and discipline, gives the attackers an overwhelming advantage. `

Reviewer: ```suggestion
			`General Choot'k cuts open the doors of a large hangar, and your ships land. The two hundred Syndicate operatives pour out wearing mechanized battle suits with armor-plated exoskeletons and weapons larger than a human could carry. They storm through the complex and you follow in their wake. Their guns melt away meter-thick walls in seconds, and most of their opposition lacks protection against the suits' stun gas bombs. With pirates focusing their fire on the imposing suits, the Unfettered make many attacks of opportunity, inflicting massive casualties. The pirates outnumber the attackers ten to one, but this huge advantage in equipment, training, and discipline, gives the attackers an overwhelming advantage.`
```

> 			`General Choot'k cuts open the doors of a large hangar, and your ships land. The two hundred Syndicate operatives pour out wearing mechanized battle suits with armor-plated exoskeletons and weapons larger than a human could carry. They storm through the complex and you follow in their wake. Their guns melt away meter-thick walls in seconds, and most of their opposition lacks protection against the suits' stun gas bombs. With pirates focusing their fire on the imposing suits, the Unfettered make many attacks of opportunity, inflicting massive casualties. The pirates outnumber the attackers ten to one, but this huge advantage in equipment, training, and discipline, gives the attackers an overwhelming advantage.`
> 			`	The squads move room to room, neutralizing all opposition with ease until their opponents switch tactics. The pirates begin using electromagnetic pulse devices to shut down the Syndicate suits (and all other electronics nearby), leaving affected operatives helpless for ten seconds. Here some of General Choot'k's Unfettered show their strength, outmaneuvering the pirates long enough for the suits to recharge. They dart between operatives and bound off walls with pulse rifles and swords. They strike rapidly and in an organized manner, before withdrawing as the operatives suits re-engage, preventing the pirates from coordinating a counter-assault.`

Reviewer 3: ```suggestion
			`General Choot'k cuts open the doors of a large hangar, and your ships land. The two hundred Syndicate operatives pour out, wearing mechanized battle suits with armor-plated exoskeletons, and armed with weapons larger than a human could carry. They storm through the complex and you follow in their wake. Their guns melt away meter-thick walls in seconds, and most of their opposition lacks protection against the suits' stun gas bombs. With pirates focusing their fire on the imposing suits, the Unfettered make many attacks of opportunity, inflicting massive casualties. The pirates outnumber the attackers ten to one, but this huge advantage in equipment, training, and discipline, gives the attackers an overwhelming advantage.`
			`	The squads move room to room, neutralizing all opposition with ease, until their opponents switch tactics. The pirates begin using electromagnetic pulse devices to shut down the Syndicate suits (and all other electronics nearby), leaving affected operatives helpless for ten long seconds. Here, some of General Choot'k's Unfettered show their strength, outmaneuvering the pirates long enough for the suits to recharge. They dart between operatives and bound off walls with pulse rifles and swords. They strike rapidly and in an organized manner, before withdrawing as the operatives suits re-engage, preventing the pirates from coordinating a counter-assault.`
```
10 seconds is a long time in a fight.

> 			`	After a bloody struggle, forty-three operatives reach the end of the fight, clearing the way to a blast door. Only a couple dozen operatives are known to be seriously wounded, but many are missing in action because the pirates were very good at trapping the suits. The Unfettered fared comparatively better, as the pirates focused most attacks on the Syndicate mechs, letting the fast, maneuverable Hai pass through. Still, over a dozen of them will take their own tails home as trophies after a pirates used miner's cutting power-arm extensions to great effect.`

Reviewer 3: ```suggestion
			`	After a bloody struggle, forty-three operatives reach the end of the fight, clearing the way to a blast door. Only a couple dozen operatives are known to be seriously wounded, but many are missing in action because the pirates were very good at trapping the suits. The Unfettered fared comparatively better, as the pirates focused most attacks on the Syndicate mechs, letting the fast, maneuverable Hai pass through. Still, over a dozen of them will take their own tails home as trophies, after the pirates used a miner's cutting power-arm extensions to great effect.`
```

> 			`	After a bloody struggle, forty-three surviving operatives win the fight, clearing the way to a blast door. Only a couple dozen operatives are critically or fatally wounded, but the pirates were very good at trapping the suits. The Unfettered fared comparatively better, having mostly acted as reserve and flanking forces, and they were evidently better trained, and acting together, supporting those in unfavorable positions and withdrawing when the time is right; still, over a dozen of them will take their own tails home as trophies after a pirates used miner's cutting power-arm extensions to great effect.`

Reviewer: ```suggestion
			`	After a bloody struggle, forty-three surviving operatives win the fight, clearing the way to a blast door. Only a couple dozen operatives are critically or fatally wounded, but the pirates were very good at trapping the suits. The Unfettered fared comparatively better, due to their tactics and good cooperation; still, over a dozen of them will take their own tails home as trophies after a pirates used miner's cutting power-arm extensions to great effect.`
```
Shorter!

> 			`	The blast door that the force now stands before is twelve meters high and wide, and scanners show it to be six meters deep. It is designed to be closed from this side though, and has not been sealed well against being reopened from this side. Soon it grinds part-way open, evidently sabotaged, and the operatives put on their goggles and pass through into the darkness of the next room.`

Reviewer 3: ```suggestion
			`	The blast door that the force now stands before is twelve meters high and wide, and scanners show it to be six meters deep. It is designed to be closed from this side, though, and has not been sealed well against being reopened on the wrong side. Soon it grinds part-way open, evidently sabotaged, and the operatives put on their goggles and pass through into the darkness of the next room.`
```

Author: I find "on the wrong side" to be more confusing than "from this side" even if it is less repetitive.

> 			`	You could be forgiven for hesitating to follow them, but General Choot'k appears beside you and places one strong paw on your shoulder. "Come, now we discover if our task is finished, or impossible. It is time for bravery." He guides you in after the operatives along with his own troops.`

Reviewer 3: ```suggestion
			`	You could be forgiven for hesitating to follow them, but General Choot'k appears beside you and places one strong paw on your shoulder. "Come, now. We will discover if our task is finished, or impossible. It is time for bravery." He guides you in after the operatives, along with his own troops.`
```

> 			`	After a bloody struggle, forty-three operatives reach the end of the fight, clearing the way to a blast door. Only a couple dozen operatives are known to be seriously wounded, but many are missing in action because the pirates were very good at trapping the suits. The Unfettered fared comparatively better, as the pirates focused most attacks on the Syndicate mechs, letting the fast, maneuverable Hai pass through. Still, over a dozen of them will take their own tails home as trophies, after the pirates used a miner's cutting power-arm extensions to great effect.`

Author: I used "pirates' devices" to avoid confusion. EMPs would be better, but I'd rather not define an acronym in the text because that would spoil the mood.

> 			`	After a bloody struggle, forty-three operatives reach the end of the fight, clearing the way to a blast door. Only a couple dozen operatives are known to be seriously wounded, but many are missing in action because the pirates were very good at trapping the suits. The Unfettered fared comparatively better, as the pirates focused most attacks on the Syndicate mechs, letting the fast, maneuverable Hai pass through. Still, over a dozen of them will take their own tails home as trophies, after the pirates used a miner's cutting power-arm extensions to great effect.`

Reviewer 4: Makes sense, feel free to resolve.

			`	"The pirates must defend this planet against a space-based lifeform that attacks in flocks. Our ships gave them a significant advantage. Fortunately, we have one more left to defend against any threats." The Korath eyes the Republic operatives as he waits for the translator to catch up.`
			`	"I must ask you keep the presence of Korath here a secret. If the Archons or other Exiles hear of this place, they may attack to retrieve us, or the Exiles may simply pillage. We will remain here in hiding unless the Exiles finally learn to settle on their planet."`
				goto questions
			label "translation services"
			action
				set "Hai Reveal [B13] Freedom: label translation services"
			`	The Korath leader hisses. "No, we cannot. We must remain here in secret. Perhaps in time there will be a way, but not yet."`
				goto questions
			label "cadmium"
			action
				set "Hai Reveal [B13] Freedom: label cadmium"
			`	The Korath leader looks to the Hai, who responds, "Is that how you were led back here?"`
			`	"Indirectly," you reply.`

> 			`	The Hai leader continues, "Fascinating, we always wondered if it would pique someone's attention! Our Korath friends here brought a ship with remarkable technology called a "Systems Core". None of the surviving Korath had the knowledge to replicate it, but they were asked to try and many of them have an uncanny knack for engineering. With their help, we built some of our own in a fashion. Being not quite right, they work, but they fuse irrevocably with the ship, and some flaw in their function causes cadmium particulate, which is shed by the system maintenance processes, into the air where it is recirculated and ignored by the system intended to filter it out, since it's part of it."`

Reviewer 3: ```suggestion
			`	The Hai leader continues, "Fascinating, we always wondered if it would pique someone's attention! Our Korath friends have ships equipped with these remarkable "Systems Cores", which are apparently vital to all Korath ships. None of the surviving Korath had the knowledge to replicate it, but they were asked to try and many of them have an uncanny knack for engineering. With their help, we built some of our own in a fashion. Being not quite right, they work, but they fuse irrevocably with the ship, and some flaw in their function causes cadmium particulate. The cadmium is shed by the system maintenance processes into the air, where it is recirculated, and ignored by the system intended to filter it out since it's a part of it."`
```
That last sentence was hard to speak out.

> 			`	The Hai leader continues, "Fascinating, we always wondered if it would pique someone's attention! Our Korath friends have ships equipped with these remarkable "Systems Cores", which are apparently vital to all Korath ships. None of the surviving Korath had the knowledge to replicate it, but they were asked to try and many of them have an uncanny knack for engineering. With their help, we built some of our own in a fashion. Being not quite right, they work, but they fuse irrevocably with the ship, and some flaw in their function causes cadmium particulate. The cadmium is shed by the system maintenance processes into the air, where it is recirculated, and ignored by the system intended to filter it out since it's a part of it."`

Reviewer 4: ```suggestion
			`	The Hai leader continues, "Fascinating, we always wondered if it would pique someone's attention! Our Korath friends have ships equipped with these remarkable "Systems Cores", which are apparently vital to all Korath ships. None of the surviving Korath had the knowledge to replicate it, but they were asked to try and many of them have an uncanny knack for engineering. With their help, we built some of our own in a fashion. Being not quite right, they work, but they fuse irrevocably with the ship, and some flaw in their function causes cadmium particulate. The cadmium is shed by the system maintenance processes into the air, where it is recirculated, and ignored by the system intended to filter it out since it's a part of the system itself."`
```

			`	The human leader adds, "We worked out how to reduce that effect to safe levels, but kept that information to ourselves in the hope that it would draw some attention. It tended to make the pirates sick more often than anyone else, which also seemed to mean that more of their victims escaped."`
				goto questions
			label home
			action
				set "Hai Reveal [B13] Freedom: label home"

> 			`	The three leaders laugh. The Hai answers, "We are home. Most of us were born here, or have lived most of our adult lives here. This is a place of learning, where three races join as one to progress together. Even some of the more recent arrivals, several had few connections back in Hai space and have viewed this place as a grand new novelty. An adventure without compare. Some recent kidnapping victims wish to leave of course, and we will help you find them and send them home."`

Reviewer 3: ```suggestion
			`	The three leaders laugh. The Hai answers, "We are home. Most of us were born here, or have lived most of our adult lives here. This is a place of learning, where three races join as one to progress together. Even some of the more recent arrivals, several had few connections back in Hai space and have viewed this place as a grand new novelty. An adventure without compare. Some recent kidnapping victims wish to leave, of course, and we will help you find them and send them home."`
```

> 			`	The three leaders laugh, and the Hai answers. "We are home. Most of us were born here, or have lived most of our adult lives here. This is a place of learning, where three races join as one to progress together. Even with several of the new arrivals had few connections back in Hai space and have viewed this place as a grand new novelty. An adventure without compare. Some recent kidnapping victims wish to leave, of course, and we will help you find them and send them home."`

Reviewer 4: ```suggestion
			`	The three leaders laugh, and the Hai answers. "We are home. Most of us were born here, or have lived most of our adult lives here. This is a place of learning, where three races join as one to progress together. Even several of the new arrivals had few connections back in Hai space and have viewed this place as a grand new novelty. An adventure without compare. Some recent kidnapping victims wish to leave, of course, and we will help you find them and send them home."`
```

> 			`	The three leaders laugh. The Hai answers, "We are home. Most of us were born here, or have lived most of our adult lives here. This is a place of learning, where three races join as one to progress together. Even some of the more recent arrivals, several had few connections back in Hai space and have viewed this place as a grand new novelty. An adventure without compare. Some recent kidnapping victims wish to leave, of course, and we will help you find them and send them home."`

Reviewer 4: ```suggestion
			`	The three leaders laugh, and the Hai answers. "We are home. Most of us were born here, or have lived most of our adult lives here. This is a place of learning, where three races join as one to progress together. Even with several of the new arrivals had few connections back in Hai space and have viewed this place as a grand new novelty. An adventure without compare. Some recent kidnapping victims wish to leave, of course, and we will help you find them and send them home."`
```

			`	The Korath speaks next. "Where my people came from, everyone was starving, barely subsisting. We had to raid for supplies, or slave away in bleak factory stations. This is a much better place for us."`
			`	The human leader concludes, "Most of the humans in this city have been here for several generations. This has always been home to us."`
				goto questions
			label place
			action
				set "Hai Reveal [B13] Freedom: label place"

> 			`	"It must seem strange to you," the human replies, "but I shall explain. This place was founded over a hundred twenty years ago by anarchists from the colony of Freedom. Their community had been working within the Syndicate for a long time, evolving into anarcho-syndicalists that worked to bring the Syndicate down from within. At some point they discovered this place after stealing a drive and other paraphernalia that were unearthed from a construction site on Foundry. With the Syndicate on its doorstep, and difficult access requirements, they saw the potential in this place, but needed more manpower to exploit it. To do this, they allied with less peaceful anarchists and pirates. Those less savory types kidnapped researchers and engineers, forcing them to expand the base, and devise new weapons or medicines, and who eventually inherited control here after the syndicalists were wiped out. Our city also has manufacturing facilities where we created goods the pirates could not easily build in human space, where their facilities are destroyed regularly.`

Reviewer 3: ```suggestion
			`	"It must seem strange to you," the human replies, "but I shall explain. This place was founded over a hundred and twenty years ago by anarchists from the colony of Freedom. Their community had been working within the Syndicate for a long time, evolving into anarcho-syndicalists that worked to bring the Syndicate down from within. At some point they discovered this place after stealing a drive and other paraphernalia that were unearthed from a construction site on Foundry. With the Syndicate on its doorstep, and difficult access requirements, they saw the potential in this place, but needed more manpower to exploit it. To do this, they allied with less peaceful anarchists and pirates. Those less savory types kidnapped researchers and engineers, forcing them to expand the base, and devise new weapons or medicines, and who eventually inherited control here after the syndicalists were wiped out. Our city also has manufacturing facilities where we created goods the pirates could not easily build in human space, where their facilities are destroyed regularly.`
```
Hundred twenty isn't wrong, but hundred and twenty works better to me.

> 			`	"It must seem strange to you," the human replies, "but I shall explain. This place was founded over a hundred and twenty years ago by anarchists from the colony of Freedom. Their community had been working within the Syndicate for a long time, evolving into anarcho-syndicalists that worked to bring the Syndicate down from within. At some point they discovered this place after stealing a drive and other paraphernalia that were unearthed from a construction site on Foundry. With the Syndicate on its doorstep, and difficult access requirements, they saw the potential in this place, but needed more manpower to exploit it. To do this, they allied with less peaceful anarchists and pirates. Those less savory types kidnapped researchers and engineers, forcing them to expand the base, and devise new weapons or medicines, and who eventually inherited control here after the syndicalists were wiped out. Our city also has manufacturing facilities where we created goods the pirates could not easily build in human space, where their facilities are destroyed regularly.`

Reviewer 4: ```suggestion
			`	"It must seem strange to you," the human replies, "but I shall explain. This place was founded over a hundred and twenty years ago by anarchists from the colony of Freedom. Their community had been working within the Syndicate for a long time, evolving into anarcho-syndicalists that worked to bring the Syndicate down from within. At some point they discovered this place after stealing a drive and other paraphernalia that were unearthed from a construction site on Foundry. With the Syndicate on its doorstep, and difficult access requirements, they saw the potential in this place, but needed more manpower to exploit it. To do this, they allied with less peaceful anarchists and pirates. Those less savory types kidnapped researchers and engineers, forcing them to expand the base and devise new weapons or medicines. They eventually inherited control here after the syndicalists were wiped out. Our city also has manufacturing facilities where we created goods the pirates could not easily build in human space, where their facilities are destroyed regularly.`
```

			`	"Now that the pirates are gone, us civilians may govern without fear, and for that you have our gratitude. As expendable upper regions of the spaceport have already been destroyed, we have little to fear from attack by space creatures in the short-term, and soon we'll have our shipyards and facilities at full capacity again, such that we may defend ourselves.`
			`	"We won't ever join the Republic or any other government beyond the wormhole, but you personally will always be welcome, along with those you trust, for what you have done today. Our research and manufacturing facilities are damaged now, but as they are repaired, we will make them available to you."`
				goto questions
			label do
			action
				set "Hai Reveal [B13] Freedom: label do"

> 			`	"We're scientists, engineers, philosophers, people of learning. Intergenerational traditions of knowledge and learning are strong here. There are laboratories and workshops throughout this city where people from all three races work together. We seek knowledge. Under the pirates, we were allowed this freedom, as they realized it was the most effective way to get results that helped them. Now that our most recent abusive masters are gone, our research will not be guided by their violent motives."`

Reviewer 4: ```suggestion
			`	"We're scientists, engineers, philosophers, people of learning. Intergenerational traditions of knowledge and scholarship are strong here. There are laboratories and workshops throughout this city where people from all three races work together. We seek knowledge. Under the pirates, we were allowed this freedom, as they realized it was the most effective way to get results that helped them. Now that our most recent abusive masters are gone, our research will not be guided by their violent motives."`
```

			`	"The planet below is a mystery to us. It has a thousand times the mass it realistically should, and its gravitational field reads as such from a great distance. This tremendous gravitational field sustains the large asteroid belt. Despite its incredible mass, as you approach the surface, the field does not increase at anywhere near the rate that it should. Thus, when you stand upon it, the gravity is only similar to that of Earth. There are many forms of radiation and strange materials inside the planet's caves. We believe it is a relic from a forgotten past, but we have no idea what sort of relic it is. Perhaps, now that we are not developing pirate weapons, we can focus on exploration."`
				goto questions
			label leave

> 			`	After several hours of discussion, during which agent Kopkow prepares the documents for the provisional recognition of the new "Elenctic Commune" with Admiral Danforth and Captain Wallace as witnesses, the wounded are actually attended to in the Elenctic hospital facilities, and most of the human operatives pack up their gear and return to the Auxiliary. Meanwhile Teeneep and Choot'k go to lengths to engage with the Hai here and discover what they would best like to do.`

Reviewer 3: ```suggestion
			`	After several hours of discussion, during which Agent Kopkow prepares the documents for the provisional recognition of the new "Elenctic Commune" with Admiral Danforth and Captain Wallace as witnesses, the wounded are actually attended to in the Elenctic hospital facilities, and most of the human operatives pack up their gear and return to the Auxiliary. Meanwhile, Teeneep and Choot'k go to lengths to engage with the Hai here and discover what they would best like to do.`
```

> 			`	After several hours of discussion, during which Agent Kopkow prepares the documents for the provisional recognition of the new "Elenctic Commune" with Admiral Danforth and Captain Wallace as witnesses, the wounded are actually attended to in the Elenctic hospital facilities, and most of the human operatives pack up their gear and return to the Auxiliary. Meanwhile, Teeneep and Choot'k go to lengths to engage with the Hai here and discover what they would best like to do.`

Reviewer 4: ```suggestion
			`	After several hours of discussion, Agent Kopkow prepares the documents for the provisional recognition of the new "Elenctic Commune" with Admiral Danforth and Captain Wallace as witnesses. The wounded are finally attended to in the Elenctic hospital facilities, and most of the human operatives pack up their gear and return to the Auxiliary. Meanwhile, Teeneep and Choot'k go to lengths to engage with the Hai here and discover what they would best like to do.`
```

			`	There is just one matter which need to be addressed with the three representatives and you raise it with them. "Did we have all of the leaders of this group? Are they all accounted for?"`
			`	They consult with some of their people who have been scouring the spaceport, helping to account for both the living, and the dead on both sides. At length, just the human comes to you.`

> 			`	"Your question was wise, there is one unaccounted for. Before I tell you though I think perhaps you should know more than you do." They reach up and push back their hood, revealing a bald head, as they gesture to the city in general. To your surprise you realize for the first time that they are a woman, though that seems like it could be the least important part of their identity.`

Reviewer 3: ```suggestion
			`	"Your question was wise, there is one unaccounted for. Before I tell you, though, I think perhaps you should know more than you do." They reach up and push back their hood, revealing a bald head, as they gesture to the city in general. To your surprise you realize for the first time that they are a woman, though that seems like it could be the least important part of their identity.`
```

> 			`	"Your question was wise, there is one unaccounted for. Before I tell you, though, I think perhaps you should know more than you do." They reach up and push back their hood, revealing a bald head, as they gesture to the city in general. To your surprise you realize for the first time that they are a woman, though that seems like it could be the least important part of their identity.`

Reviewer 4: ```suggestion
			`	"Your question was wise. There is one unaccounted for. Before I tell you, though, I think perhaps you should know more than you do." They reach up and push back their hood, revealing a bald head, as they gesture to the city in general. To your surprise you realize for the first time that they are a woman, though that seems like it could be the least important part of their identity.`
```

			choice
				`	"What do you wish to tell me?"`
				`	"I've had enough history lessons for one day. Can you tell me the leader's name?"`
					goto impatient

> 			`	"Many different overlords we have had, and always they have relied on recruits from the city for some capacities, and have rallied other lesser groups to their banner. As such, ever have their been factions. Some, kinder than others."`

Reviewer 4: ```suggestion
			`	"Many different overlords we have had, and always they have relied on recruits from the city for some capacities, and have rallied other lesser groups to their banner. As such, ever have there been factions. Some, kinder than others."`
```

			`	"I tell you all this because I suspect you must seek out this one leader who is unaccounted for, and you should do so with all the knowledge at your disposal. The one leader who is not here was the last major player from the new 'Base Principles'. I do not know where your loyalties lie, so I ask you do you wish to know?"`
				goto reason
			label impatient

> 			`	They sigh. "The one leader who is not here was the last major player from the new 'Base Principles'. I do not know where your loyalties lie, so I ask you do you wish to know?"`

Reviewer 4: ```suggestion
			`	They sigh. "The one leader who is not here was the last major player from the new 'Base Principles'. I do not know where your loyalties lie, so I ask you: do you wish to know?"`
```

> 			`	"I tell you all this because I suspect you must seek out this one leader who is unaccounted for, and you should do so with all the knowledge at your disposal. The one leader who is not here was the last major player from the new 'Base Principles'. I do not know where your loyalties lie, so I ask you: why do you wish to know?"`

Author: This should also be "why do you wish to know." The speaker is asking Captain Last the reason for their interest in the information..

> 			`	They sigh. "The one leader who is not here was the last major player from the new 'Base Principles'. I do not know where your loyalties lie, so I ask you do you wish to know?"`

Reviewer 4: ```suggestion
			`	They sigh. "The one leader who is not here was the last major player from the new 'Base Principles'. I do not know where your loyalties lie, so I ask you: do you wish to know?"`
```

Author: This should be "why do you wish to know"

Author: It now reads, "I ask you: why do you wish to know"

			`	They lift their head and smile. "A promise made is a promise kept. If your word is your bond then it would not do for me to stand in its way.`
				goto finished
			label finished
			`	"The missing leader you seek is embedded within the Syndicate and is known by the name Terry Adrianopoulos."`
			choice
				`	"Terry did this!?"`
					goto surprised
				`	"I suspected as much."`
			`	Their words scratch an itch of suspicion that you didn't realize had been irritating you until now. They smirk knowingly. "I have merely confirmed that which you already guessed. Very well, I need say no more. This matter is yours to do with as you will." They leave you to ponder your course.`
				goto done
			label surprised
			`	They smile at you with subtle mirth at your surprise. "I see you know of this person, and did not suspect. I think it is to you and you alone I shall impart this knowledge. Do with it as you will." They leave you to ponder your course.`
			label done
			`	As you return to your ship, Teeneep interrupts you. "I need you to escort me one last time, Captain <last>. I must return to <planet> to resolve things now that the threat is ended. You will be well-paid, of course."`

> 			`	Anticipating your next question, she says, "The ships of the Unfettered Hai will be escorted back with us since the Elenctics are convinced they are at no risk and still have a ship at their disposal. Some of those who come as crew on the Unfettered vessels have chosen to remain here and join the Elenctics. More than enough bunks have subsequently been freed to return those who have chosen to come home with us. Danforth will come with us, but the other human vessels will make their way home themselves. Now we must rejoin our people on <planet> and set about resolving this matter for good, finally."`

Reviewer 3: ```suggestion
			`	Anticipating your next question, she says, "The ships of the Unfettered Hai will be escorted back with us, since the Elenctics are convinced they are at no risk and still have a ship at their disposal. Some of those who come as crew on the Unfettered vessels have chosen to remain here and join the Elenctics. More than enough bunks have subsequently been freed to return those who have chosen to come home with us. Danforth will come with us, but the other human vessels will make their way home themselves. Now we must rejoin our people on <planet> and set about resolving this matter for good, finally."`
```

				goto finished
			label knowhai
			`	They incline their head respectfully. "The Hai are perhaps the greatest valuers of truth among us. If they have set you to this, then they will be honored by your diligence.`
				goto finished
			label knowsyndi
			`	They focus on you closely. "The Syndicate is perhaps the most impressive machine ever wrought by man. It is worthy of both respect and fear, and will do to people as any machine does to the product of its design. Treat it as such and it will always do precisely what you expect it to; both good and bad.`
				goto finished
			label knowduty
			`	They lift their head and smile. "A promise made is a promise kept. If your word is your bond then it would not do for me to stand in its way.`
				goto finished
			label finished
			`	"The missing leader you seek is embedded within the Syndicate and is known by the name Terry Adrianopoulos."`
			choice

> 				`	"Terry did this!?"`

Reviewer 4: ```suggestion
				`	"Terry did this?"`
```

Author: The player is supposed to be surprised. Removing the exclamation point eliminates the feeling of surprise.

Author: If you don't like the interrobang, then the only option is to split up the sentence: "Aghast! Terry did this?"

Author: Splitting up the sentence would reduce the feeling of surprise, and an interrobang is the correct option here. If you can come up with another way to express surprise and ask a question in few words, then I'm game for it.

Reviewer 4: I'm not really passionate about the change, I'm fine with it as is.

		log `Factions` `Elenchus` `Formerly a pirate stronghold, this city built by generations of kidnapped specialists is a hundred twenty years old now. Humans, Hai, and even Korath live peacefully, working side-by-side, and most have no desire to leave. They value their independence and have refused Republic membership. This city orbits a strange world that defies analysis in a region of space untouched by civilization.`
		log `Liberated Elenchus.`
		conversation
			`General Choot'k cuts open the doors of a large hangar, and your ships land. The two hundred Syndicate operatives pour out, wearing armor-plated exoskeletons over two meters high, and equipped with all manner of weapons and gadgetry. They storm through the complex and you follow in their wake. Their tools melt away meter-thick walls in seconds, and most of their opposition lacks protection against the suits' stun gas bombs. With pirates focusing their fire on the imposing suits, the Unfettered make many attacks of opportunity, inflicting massive casualties. The pirates outnumber the attackers ten to one, but this huge advantage in equipment, training, and discipline gives the attackers an overwhelming advantage.`

> 			`	The squads move room to room, neutralizing all opposition with ease, until their opponents switch tactics. The pirates begin using electromagnetic pulse devices to shut down the Syndicate suits (and all other electronics nearby), leaving affected operatives helpless for ten long seconds. Here, some of General Choot'k's Unfettered show their strength, outmaneuvering the pirates long enough for the suits to recharge. They dart between operatives and bound off walls with pulse rifles and swords. They strike rapidly and in an organized manner, before withdrawing as the operatives suits re-engage, preventing the pirates from coordinating a counter-assault.`

Reviewer 5: I think this is a good change with subtle but important meaning, but "fighting" is a bit redundant here. Even if a technical term, the more general "long knives" still seems most appropriate.

> 			`	The squads move room to room, neutralizing all opposition with ease, until their opponents switch tactics. The pirates begin using electromagnetic pulse devices to shut down the Syndicate suits (and all other electronics nearby), leaving affected operatives helpless for ten long seconds. Here, some of General Choot'k's Unfettered show their strength, outmaneuvering the pirates long enough for the suits to recharge. They dart between operatives and bound off walls with pulse rifles and swords. They strike rapidly and in an organized manner, before withdrawing as the operatives suits re-engage, preventing the pirates from coordinating a counter-assault.`

Author: That sounds too "technical report" to me, so I used "long blades."

> 			`	The squads move room to room, neutralizing all opposition with ease, until their opponents switch tactics. The pirates begin using electromagnetic pulse devices to shut down the Syndicate suits (and all other electronics nearby), leaving affected operatives helpless for ten long seconds. Here, some of General Choot'k's Unfettered show their strength, outmaneuvering the pirates long enough for the suits to recharge. They dart between operatives and bound off walls with pulse rifles and swords. They strike rapidly and in an organized manner, before withdrawing as the operatives suits re-engage, preventing the pirates from coordinating a counter-assault.`

Reviewer 6: long blades is like a sword - at least as long as your leg
long knives is explicitly shorter than a sword - more like the length of your forearm

They have *very* different implications.

> 			`	After a bloody struggle, forty-three operatives reach the end of the fight, clearing the way to a blast door. Only a couple dozen operatives are known to be seriously wounded, but many are missing in action because the pirates' devices were very effective at trapping the suits. The Unfettered fared comparatively better, as the pirates focused most attacks on the Syndicate mechs, letting the fast, maneuverable Hai pass through. Still, over a dozen of them will take their own tails home as trophies, after the pirates used a miner's cutting power-arm extensions to great effect.`

Reviewer 6: Without a clarifying explanation there's no actual reason presented for how suits that are temporarily disabled by EMP could result in MIA operatives.
```suggestion
			`	After a bloody struggle, forty-three operatives reach the end of the fight, clearing the way to a blast door. Only a couple dozen operatives are known to be seriously wounded, but many were disabled after the pirates' began using traps and machinery to pin, crush, or disable operatives or their suits. The Unfettered fared comparatively better, as the pirates focused most attacks on the Syndicate operatives, letting the fast, maneuverable Hai pass through. Still, over a dozen of them will take their own tails home as trophies, after the pirates used a miner's cutting power-arm extensions to great effect.`
```

Reviewer 5: ```suggestion
			`	After a bloody struggle, forty-three operatives reach the end of the fight, clearing the way to a blast door. Only a couple dozen operatives are known to be seriously wounded, but many have been missing since the pirates switched tactics and began using traps and machinery to pin, crush, or disable the operatives or their suits. The Unfettered fared comparatively better, as the pirates focused most attacks on the Syndicate operatives, letting the fast, maneuverable Hai pass through. Still, over a dozen of them will take their own tails home as trophies, after the pirates used a miner's cutting power-arm extensions to great effect.`
```

Bit of suggested addition to @MasterOfGrey 's above.

> 			`	The blast door that the force now stands before is twelve meters high and wide, and scanners show it to be six meters deep. It is designed to be closed from this side, though, and has not been sealed well against being reopened. Soon it grinds part-way open, evidently sabotaged, and the operatives put on their goggles and pass through into the darkness of the next room.`

Reviewer 5: ```suggestion
			`	The blast door that the force now stands before is twelve meters high and wide, and scanners show it to be six meters deep. It is designed to be closed from this side, though, and has not been sealed well against being reopened. Soon it grinds open but, evidently sabotaged, only part-way. Despite this attempt, the operatives put on their goggles and easily pass through into the darkness of the next room.`
```

> 			`	You could be forgiven for hesitating to follow them, but General Choot'k appears beside you and places one strong paw on your shoulder. "Come, now. We will discover if our task is finished, or impossible. It is time for bravery." He guides you in after the operatives, along with his own troops.`

Reviewer 5: ```suggestion
			`	You could be forgiven for hesitating to follow them, but General Choot'k appears beside you and places one strong paw on your shoulder. "Come, now. We will discover if our task is finished or impossible. It is time for bravery." He guides you in after the operatives, along with his own troops.`
```

			choice
				`	"Lead on."`
					goto leadon

> 				`	"Are you calling me a coward? If I had more time, I'd challenge you to a duel for that."`

Reviewer 5: ```suggestion
				`	"I'm no coward; such accusation is challenge to a duel."`
```

Attempted shortening for conversation below.

Author: That sounds too lawyery to me, so I used, "Treating me like a coward? That's grounds for a duel."

			`	"Heh heh," he laughs in a low voice. "Maybe soon."`
			action

> 				set "Hai Reveal [B13] Freedom: threatened Choot'k"

Reviewer 6: @Hurleveur do you have a use-case in mind for this?
I ask only because this choice here feels somewhat gratuitous, and the player's response feels somewhat out of step with everything else so far.

Reviewer: It could fall more into character but I'm the one that asked for it to set a condition so it could have consequences, yeah I was thinking a duel but not right now, like when you either get the Emerald Sword or later on when he's thinking of upgrading to a Roach (and thus lets you capture the ship) maybe both, but that's without this happening.

It's true this does seem a bit out of step... If it stays I'd probably add a duel at the end of the campaign (provided he gets his Emperor Beetle, ofc) without requiring to have an Emerald Sword

Reviewer 6: :thinking face: I don't actually mind the choice being here, but the "threat" option needs to be drastically less verbose.

Author: I used, "Treating me like a coward? That's grounds for a duel."

> 				set "Hai Reveal [B13] Freedom: threatened Choot'k"

Reviewer 5: ```suggestion
				set "Hai Reveal [B13] Freedom: challenged Choot'k"
```
Even with the original phrasing, this is far more a challenge than a threat, and this situation is one where the difference is quite important.

			label leadon

> 			`	Scanners show the room is a vast cavern, some sort of public space suitable for ten thousand people or more. It's pitch black when you enter, but as the last of the troops file in, a second blast door closes behind you. High roof lights snap on making the room as bright as day in the blink of an eye.`

Reviewer 5: ```suggestion
			`	It's pitch black when you enter, but scanners show the room is a vast cavern, some sort of public space suitable for ten thousand people or more. As the last of the troops file in, a second blast door closes behind you. High roof lights snap on making the room as bright as day in the blink of an eye.`
```

> 			`	You see before you a couple hundred pirates, held captive by a large crowd of heavily-armed civilians - all of which are hesitantly pointing guns at the operatives. It seems the citizens arranged a violent rebellion to coincide with the Republic's attack. Three unarmed civilians wearing distinctive hooded robes walk towards the operatives: one Hai, one human, and to your surprise one lizard-like Korath. The leader of the Syndicate operatives exits her suit and approaches them.`

Reviewer 6: It has occurred to me that it can no longer be assumed by this campaign that you know what a Korath is. However, given that there are now systems beyond Stormhold that are raided/mined by Exiles, maybe we should include some kind of brief earlier during one of the Stormhold stops about the Korath and the (perhaps surprising) amount of Korath equipment that the Republic found once they occupied it?

Reviewer 5: That would probably be a good addition.

Author: If someone can give me a bit more information on what each group is supposed to know about the Korath at Stormhold, I can add that.

Should that be in this PR or another one?

> 			`	You see before you a couple hundred pirates, held captive by a large crowd of heavily-armed civilians - all of which are hesitantly pointing guns at the operatives. It seems the citizens arranged a violent rebellion to coincide with the Republic's attack. Three unarmed civilians wearing distinctive hooded robes walk towards the operatives: one Hai, one human, and to your surprise one lizard-like Korath. The leader of the Syndicate operatives exits her suit and approaches them.`

Reviewer 5: ```suggestion
			`	You see before you a couple hundred pirates, held captive by a large crowd of heavily-armed civilians - many of which now hesitantly point their guns at the operatives instead. It seems the citizens arranged a violent rebellion to coincide with the Republic's attack. Three unarmed civilians wearing distinctive hooded robes walk towards the operatives: one Hai, one human, and to your surprise one lizard-like Korath. The leader of the Syndicate operatives exits her suit and approaches them.`
```

> 			`	You see before you a couple hundred pirates, held captive by a large crowd of heavily-armed civilians - all of which are hesitantly pointing guns at the operatives. It seems the citizens arranged a violent rebellion to coincide with the Republic's attack. Three unarmed civilians wearing distinctive hooded robes walk towards the operatives: one Hai, one human, and to your surprise one lizard-like Korath. The leader of the Syndicate operatives exits her suit and approaches them.`

Reviewer 5: ```suggestion
			`	You see before you scores of pirates, corralled and held captive by an equally large crowd of heavily-armed civilians - all of which are hesitantly pointing guns at the operatives. It seems the citizens arranged a violent rebellion to coincide with the Republic's attack. Three unarmed civilians wearing distinctive hooded robes walk towards the operatives: one Hai, one human, and to your surprise one lizard-like Korath. The leader of the Syndicate operatives exits her suit and approaches them.`
```

Bit more extensive change to tone here to make the placement of groups a bit clearer imo

> 			`	The Hai speaks. "We are the leaders of the civilian government of Elenchus. Although we are grateful for your assistance in defeating our violent rulers, we will not accept your dominion either. We have heard of your civil war. Elenchus is an independent nation; and we intend to stay so unless you intend to force otherwise. I can, however, promise that unlike our overlords, we have no interest in anything other than peace, and we will send no military ships through the wormhole.`

Reviewer 5: ```suggestion
			`	The Hai speaks. "We are the leaders of the civilian government of Elenchus. Although we are grateful for your assistance in defeating our violent rulers, we will not accept your dominion either. We have heard of your civil war. Elenchus is an independent nation, and we intend to stay so unless you intend to force otherwise. I can, however, promise that unlike our overlords, we have no interest in anything other than peace. We will send no military ships through the wormhole.`
```

> 			`	"Captain <last>, we understand our former slave masters agreed to pay tribute to you. We will continue these payments out of gratitude for ending our servitude."`

Reviewer 6: This behaviour is unctuous, and should not be brought up right here.
I would have them bring it up later, or maybe have you bring it up in a question.

Lorewise, this payment could only be being made if your ship's systems now carried some sort of approval code for a transaction to be set up when you reconnected to human space and the institutions where the pirates surreptitiously controlled accounts that they used. There'd be no way for you to actually be credited from here directly.
So it could be that you tell them about it and that gives them the information needed to actually gain access to those accounts, presumably via some sort of offline "wallet" that would be on the system here. At which point you could continue to receive that payment as a stipend; partly out of gratitude, and partly to encourage you to visit again occasionally. (If the money stopped you'd know someone else had gained control of the accounts.)

If we don't want to do a lore dive for it though I'd suggest simply removing it.

Reviewer 5: I am still thinking that the tributing aspect may not be the best end place for the mission at all, but a proper in game justification for it's involvement would definitely require direct reference to be this consistent and with a plot progression tie in.

Author: I removed the tribute bit. It'll have to stay unresolved for a while.

> 			`	"Captain <last>, we understand our former slave masters agreed to pay tribute to you. We will continue these payments out of gratitude for ending our servitude."`

Reviewer 5: ```suggestion
```

I would highly second/suggest just removing this line. Replacement per Grey's comment above can come later as another focused PR if approriate.

			choice
				`	"Thank you. I have a few questions for you before I go."`

> 				`	"I wish you a successful future with your newfound freedom. For now, it is time for me to depart."`

Reviewer 6: You are in the middle of resolving a major diplomatic incident with multiple government representatives with you. You cannot simply go from barging through their doors with armed troops to asking them questions as your individual person.
`	After several hours of discussion, during which agent Kopkow prepares the documents for the provisional recognition of the new "Elenctic Commune" with Admiral Danforth and Captain Wallace as witnesses, the wounded are actually attended to in the Elenctic hospital facilities, and most of the human operatives pack up their gear and return to the Auxiliary. Meanwhile Teeneep and Choot'k go to lengths to engage with the Hai here and discover what they would best like to do.`
This (or some equivalent) is an important line to occur *before* any great discussion you can have with those reps.

Author: I moved it back up. One later paragraph needed a little tweak to get it to read well with that change.

				`	"Can you speak to other Korath for me?"`
					goto "translation services"
					to display
						not "Hai Reveal [B13] Freedom: label translation services"
						has "Hai Reveal [B13] Freedom: label korath"
				`	"I wish you a successful future with your newfound freedom. For now, it is time for me to depart."`
			`	"It was a pleasure meeting you," says the human leader.`
				goto leave
			label korath
			action
				set "Hai Reveal [B13] Freedom: label korath"
			`	The Korath leader speaks, with a thick, sibilant accent. "Several generations ago, three of our raiding ships jumped to the Alcyone system. Numerous pirate vessels left Stormhold and disabled our ships. A large human ship boarded our flagship, and used what you call 'nerve gas' to kill the entire crew. The death is excruciatingly painful to Korath. The pirates broadcast the footage over to us, so we could see the screaming. Not wanting that fate, most of the raiders on the other Korath ships surrendered. We worked as slaves for the pirate gangs, but now we're here, safe, and free. We are not specialists or their descendants like the others here, but we have other talents.`

> 			`	"The pirates must defend this planet against a space-based lifeform that attacks in flocks. Our ships gave them a significant advantage. Fortunately, we have one more left to defend against any threats." The Korath eyes the Republic operatives as he waits for the translator to catch up.`

Reviewer 6: ```suggestion
			`	"The pirates must defend this planet against a space-based lifeform that attacks in flocks. Our ships gave them a significant advantage. Fortunately, we have one more left to defend against any threats." The Korath eyes the Syndicate operatives as he waits for the translator to catch up.`
```

			`	"I must ask you keep the presence of Korath here a secret. If the Archons or other Exiles hear of this place, they may attack to retrieve us, or the Exiles may simply pillage. We will remain here in hiding unless the Exiles finally learn to settle on their planet."`
				goto questions
			label "translation services"
			action
				set "Hai Reveal [B13] Freedom: label translation services"

> 			`	The Korath leader hisses. "No, we cannot. We must remain here in secret. Perhaps in time there will be a way, but not yet."`

Reviewer 6: Little bit more lore-conformance.
```suggestion
			`	The Korath leader hisses. "No, we cannot. We here do not even speak a full form of our own language anymore. Even if we did, we must remain here in secret. Perhaps in time there there will be a reconnection, but not yet."`
```

				goto questions
			label "cadmium"
			action
				set "Hai Reveal [B13] Freedom: label cadmium"
			`	The Korath leader looks to the Hai, who responds, "Is that how you were led back here?"`
			`	"Indirectly," you reply.`

> 			`	The Hai leader continues, "Fascinating, we always wondered if it would pique someone's attention! Our Korath friends have ships equipped with these remarkable "Systems Cores", which are apparently vital to all Korath ships. None of the surviving Korath had the knowledge to replicate it, but they were asked to try and many of them have an uncanny knack for engineering. With their help, we built some of our own in a fashion. Being not quite right, they work, but they fuse irrevocably with the ship, and some flaw in their function causes cadmium particulate. The cadmium is shed by the system maintenance processes into the air, where it is recirculated, and ignored by the system intended to filter it out since it's a part of the system itself."`

Reviewer 6: ```suggestion
			`	The Hai leader continues, "Fascinating, we always wondered if it would pique someone's attention! Our Korath friends have ships equipped with these remarkable "Systems Cores", which are apparently vital to all Korath ships. None of the surviving Korath had the knowledge to replicate it, but they were asked to try and many of them have an uncanny knack for engineering. With their help, we attempted to build some equivalents of our own. Being not quite right, they work, but they fuse irrevocably with the ship, and some flaw in their function causes cadmium particulate. The cadmium is shed by the system maintenance processes into the air, where it is recirculated, and ignored by the system intended to filter it out since it's a part of the system itself."`
```

				goto questions
			label home
			action
				set "Hai Reveal [B13] Freedom: label home"
			`	The three leaders laugh, and the Hai answers. "We are home. Most of us were born here, or have lived most of our adult lives here. This is a place of learning, where three races join as one to progress together. Even several of the new arrivals had few connections back in Hai space and have viewed this place as a grand new novelty. An adventure without compare. Some recent kidnapping victims wish to leave, of course, and we will help you find them and send them home."`
			`	The Korath speaks next. "Where my people came from, everyone was starving, barely subsisting. We had to raid for supplies, or slave away in bleak factory stations. This is a much better place for us."`
			`	The human leader concludes, "Most of the humans in this city have been here for several generations. This has always been home to us."`
				goto questions
			label place
			action
				set "Hai Reveal [B13] Freedom: label place"

> 			`	"It must seem strange to you," the human replies, "but I shall explain. This place was founded over a hundred and twenty years ago by anarchists from the colony of Freedom. Their community had been working within the Syndicate for a long time, evolving into anarcho-syndicalists that worked to bring the Syndicate down from within. At some point they discovered this place after stealing a drive and other paraphernalia that were unearthed from a construction site on Foundry. With the Syndicate on its doorstep, and difficult access requirements, they saw the potential in this place, but needed more manpower to exploit it. To do this, they allied with less peaceful anarchists and pirates. Those less savory types kidnapped researchers and engineers, forcing them to expand the base and devise new weapons or medicines. They eventually inherited control here after the syndicalists were wiped out. Our city also has manufacturing facilities where we created goods the pirates could not easily build in human space, where their facilities are destroyed regularly.`

Reviewer 5: ```suggestion
			`	"It must seem strange to you," the human replies, "but I shall explain. This place was founded over a hundred and twenty years ago by anarchists from the colony of Freedom. Their community had been working within the Syndicate for a long time, evolving into anarcho-syndicalists that worked to bring the Syndicate down from within. At some point they discovered this place after stealing a drive and other paraphernalia that were unearthed from a construction site on Foundry. With the Syndicate on its doorstep and difficult access requirements, they saw the potential in this place but needed more manpower to exploit it. To do this, they allied with less peaceful anarchists and pirates. Those less savory types kidnapped researchers and engineers, forcing them to expand the base and devise new weapons or medicines. They eventually inherited control here after the syndicalists were wiped out. Our city also has manufacturing facilities where we created goods the pirates could not easily build in human space, where their facilities are destroyed regularly.`
```

Reviewer 5: Not sure why the removed commas aren't highlighted here. :\

			`	"Now that the pirates are gone, us civilians may govern without fear, and for that you have our gratitude. As expendable upper regions of the spaceport have already been destroyed, we have little to fear from attack by space creatures in the short-term, and soon we'll have our shipyards and facilities at full capacity again, such that we may defend ourselves.`
			`	"We won't ever join the Republic or any other government beyond the wormhole, but you personally will always be welcome, along with those you trust, for what you have done today. Our research and manufacturing facilities are damaged now, but as they are repaired, we will make them available to you."`
				goto questions
			label do
			action
				set "Hai Reveal [B13] Freedom: label do"

> 			`	"We're scientists, engineers, philosophers, people of learning. Intergenerational traditions of knowledge and scholarship are strong here. There are laboratories and workshops throughout this city where people from all three races work together. We seek knowledge. Under the pirates, we were allowed this freedom, as they realized it was the most effective way to get results that helped them. Now that our most recent abusive masters are gone, our research will not be guided by their violent motives."`

Reviewer 6: Tweaking re feedback that it makes it sound too much like a utopia.
```suggestion
			`	"We're scientists, engineers, philosophers, people of learning. Intergenerational traditions of knowledge and scholarship are strong here. There are laboratories and workshops throughout this city where we work on all manner of things. We seek knowledge. This place is far from being a paradise, but it is a community that strives together. Under the pirates we were allowed this freedom, as they realized it was the most effective way to get results that helped them. Now that our most recent abusive masters are gone, we need not have our priorities guided by their violent motives."`
```

			`	"The planet below is a mystery to us. It has a thousand times the mass it realistically should, and its gravitational field reads as such from a great distance. This tremendous gravitational field sustains the large asteroid belt. Despite its incredible mass, as you approach the surface, the field does not increase at anywhere near the rate that it should. Thus, when you stand upon it, the gravity is only similar to that of Earth. There are many forms of radiation and strange materials inside the planet's caves. We believe it is a relic from a forgotten past, but we have no idea what sort of relic it is. Perhaps, now that we are not developing pirate weapons, we can focus on exploration."`
				goto questions
			label leave

> 			`	After several hours of discussion, Agent Kopkow prepares the documents for the provisional recognition of the new "Elenctic Commune" with Admiral Danforth and Captain Wallace as witnesses. The wounded are finally attended to in the Elenctic hospital facilities, and most of the human operatives pack up their gear and return to the Auxiliary. Meanwhile, Teeneep and Choot'k go to lengths to engage with the Hai here and discover what they would best like to do.`

Reviewer 6: As mentioned, most of this line needs to be earlier, but there is some value to having the provisional recognition be mentioned here.

Author: I moved it back.

			`	There is just one matter which need to be addressed with the three representatives and you raise it with them. "Did we have all of the leaders of this group? Are they all accounted for?"`
			`	They consult with some of their people who have been scouring the spaceport, helping to account for both the living, and the dead on both sides. At length, just the human comes to you.`
			`	"Your question was wise. There is one unaccounted for. Before I tell you, though, I think perhaps you should know more than you do." They reach up and push back their hood, revealing a bald head, as they gesture to the city in general. To your surprise you realize for the first time that they are a woman, though that seems like it could be the least important part of their identity.`
			choice
				`	"What do you wish to tell me?"`
				`	"I've had enough history lessons for one day. Can you tell me the leader's name?"`
					goto impatient

> 			`	"Many different overlords we have had, and always they have relied on recruits from the city for some capacities, and have rallied other lesser groups to their banner. As such, ever have there been factions. Some, kinder than others."`

Reviewer 5: ```suggestion
			`	"Many different overlords we have had, and always they have relied on recruits from the city for some capacities while rallying other lesser groups to their banner. As such, ever have there been factions. Some, kinder than others."`
```

		log `Factions` `Elenchus` `Formerly a pirate stronghold, this city built by generations of kidnapped specialists is a hundred twenty years old now. Humans, Hai, and even Korath live peacefully, working side-by-side, and most have no desire to leave. They value their independence and have refused Republic membership. This city orbits a strange world that defies analysis in a region of space untouched by civilization.`
		log `Liberated Elenchus.`
		conversation
			`General Choot'k cuts open the doors of a large hangar, and your ships land. The two hundred Syndicate operatives pour out, wearing exoskeletons with high density armor plating and equipped with all manner of weapons and gadgetry. They storm through the complex and you follow in their wake. Their tools melt through security locks in seconds, and most of their opposition lacks protection against the suits' stun gas bombs. With pirates focusing their efforts on the imposing suits, the Unfettered make many attacks of opportunity, inflicting massive casualties. The pirates outnumber the attackers ten to one, but this huge advantage in equipment, training, and discipline gives the attackers an overwhelming advantage.`

> 			`	The squads move room to room, neutralizing all opposition with ease, until their opponents switch tactics. The pirates begin using electromagnetic pulse devices to shut down the Syndicate suits (and all other electronics nearby), leaving affected operatives helpless for ten long seconds. Here, some of General Choot'k's Unfettered show their strength, outmaneuvering the pirates long enough for the suits to recharge. They dart between operatives and bound off walls with pulse rifles and long blades. They strike rapidly and in an organized manner, before withdrawing as the operatives suits re-engage, preventing the pirates from coordinating a counter-assault.`

Reviewer 6: As mentioned, this does need to be knives, not blades.
```suggestion
			`	The squads move room to room, neutralizing all opposition with ease, until their opponents switch tactics. The pirates begin using electromagnetic pulse devices to shut down the Syndicate suits (and all other electronics nearby), leaving affected operatives helpless for ten long seconds. Here, some of General Choot'k's Unfettered show their strength, outmaneuvering the pirates long enough for the suits to recharge. They dart between operatives and bound off walls with pulse rifles and long knives. They strike rapidly and in an organized manner, before withdrawing as the operatives suits re-engage, preventing the pirates from coordinating a counter-assault.`
```

				goto finished
			label knowsyndi
			`	They focus on you closely. "The Syndicate is perhaps the most impressive machine ever wrought by man. It is worthy of both respect and fear, and will do to people as any machine does to the product of its design. Treat it as such and it will always do precisely what you expect it to; both good and bad.`
				goto finished
			label knowduty
			`	They lift their head and smile. "A promise made is a promise kept. If your word is your bond then it would not do for me to stand in its way.`
				goto finished
			label finished
			`	"The missing leader you seek is embedded within the Syndicate and is known by the name Terry Adrianopoulos."`
			choice
				`	"Terry did this!?"`
					goto surprised
				`	"I suspected as much."`
			`	Their words scratch an itch of suspicion that you didn't realize had been irritating you until now. They smirk knowingly. "I have merely confirmed that which you already guessed. Very well, I need say no more. This matter is yours to do with as you will." They leave you to ponder your course.`
				goto done
			label surprised

> 			`	They smile at you with subtle mirth at your surprise. "I see you know of this person, and did not suspect. I think it is to you and you alone I shall impart this knowledge. Do with it as you will." They leave you to ponder your course.`

Reviewer 5: ```suggestion
			`	They smile with subtle mirth at your surprise. "I see you know of this person and did not suspect. I think it is to you, and you alone, I shall impart this knowledge. Do with it as you will." They leave you to ponder your course.`
```

> 			`	They smile at you with subtle mirth at your surprise. "I see you know of this person, and did not suspect. I think it is to you and you alone I shall impart this knowledge. Do with it as you will." They leave you to ponder your course.`

Reviewer 5: ```suggestion
			`	They smile with subtle mirth at your surprise. "I see you know of this person but did not suspect. I think it is to you, and you alone, I shall impart this knowledge. Do with it as you will." They leave you to ponder your course.`
```

This version carries a bit more accusatory weight towards Terry. It's a good place to have a subtle line as to this speaker's views of her, but I'm not sure which side of that line we want them on.

			label done
			`	As you return to your ship, Teeneep interrupts you. "I need you to escort me one last time, Captain <last>. I must return to <planet> to resolve things now that the threat is ended. You will be well-paid, of course."`

> 			`	Anticipating your next question, she says, "The ships of the Unfettered Hai will be escorted back with us, since the Elenctics are convinced they are at no risk and still have a ship at their disposal. Some of those who come as crew on the Unfettered vessels have chosen to remain here and join the Elenctics. More than enough bunks have subsequently been freed to return those who have chosen to come home with us. Danforth will come with us, but the other human vessels will make their way home themselves. Now we must rejoin our people on <planet> and set about resolving this matter for good, finally."`

Reviewer 5: ```suggestion
			`	Anticipating your next question, she says, "The ships of the Unfettered Hai will be escorted back with us, since the Elenctics are convinced they are at no risk and still have a ship at their disposal. Some of those who came as crew on the Unfettered vessels have chosen to remain here and join the Elenctics. More than enough bunks have subsequently been freed to return those who have chosen to come home with us. Danforth will come with us, but the other human vessels will make their way home themselves. Now we must rejoin our people on <planet> and set about resolving this matter for good, finally."`
```
